_assets/scripts/test-with-coverage.sh

```diff
@@ -13,7 +13,7 @@ if [[ "$*" != *"-test.run"* ]]; then
   count=${TEST_WITH_COVERAGE_COUNT}
 fi
 
-go test -json \
+env LD_LIBRARY_PATH="${LD_LIBRARY_PATH}" go test -json \
```

Comment:
This seems pointless. If it needs to be exported then export it where you set it.